### PR TITLE
Update the runtime

### PR DESCRIPTION
**Pull request by AsciiWolf**
/cc @bbhtt

### CODE REVIEW

**Comment by bbhtt** [comment]
Can you try updating zlib too to the latest version? This one is affected by multiple security issues.

Also libjxl can be removed from the manifest, GNOME 45 comes with it by default.

Thanks.

**Comment by AsciiWolf** [comment]
Build still fails on:

`ImportError: /usr/lib/python3.11/site-packages/libxml2mod.so: undefined symbol: xmlNewSAXParserCtxt`

There is a python2-libxml2 module in the manifest, but it is on a latest available version and also it is compiled with `--with-python=python2`.

@bbhtt Any idea?

File: org.xiphos.Xiphos.json [hunk 1]
```diff
@@ -44,6 +44,16 @@
...
+    {
+        "name": "python3-libxml2",
+        "sources": [
+            {
+                "type": "archive",
+                "url": "https://download.gnome.org/sources/libxml2/2.12/libxml2-2.12.4.tar.xz",
+                "sha256": "497360e423cf0bd99eacdb7c6215dea92e6d6e89ee940393c2bae0e77cb9b7d0"
+            }
+        ]
+    },
```
Comment (bbhtt) [inline org.xiphos.Xiphos.json]: This is in the runtime, try bumping python2-libxml2 to 2.11.5 to match the version in SDK https://gitlab.gnome.org/GNOME/libxml2/-/archive/v2.11.5/libxml2-v2.11.5.tar.bz2

itstool at least works with that, but webkit2gtk might complain, not possible to build webkit2gtk locally

Comment (AsciiWolf) [reply org.xiphos.Xiphos.json]: Done, thanks!

**Comment by AsciiWolf** [comment]
screenshot

Tested and seems to work fine. @bbhtt Feel free to merge. :-)

**Comment by bbhtt** [comment]
Thanks!

**Pull request merged**